morpheus-DFP/.vscode/launch.json

```diff
@@ -0,0 +1,29 @@
+{
+  "version": "0.2.0",
+  "configurations": [
+    {
+      "name": "Next.js: debug server-side",
+      "type": "node-terminal",
+      "request": "launch",
+      "command": "CUDA_VISIBLE_DEVICES=1 yarn dev"
+    },
+    {
+      "name": "Next.js: debug client-side",
+      "type": "pwa-chrome",
+      "request": "launch",
+      "url": "http://localhost:3000"
+    },
+    {
+      "name": "Next.js: debug full stack",
+      "type": "node-terminal",
+      "request": "launch",
+      "command": "CUDA_VISIBLE_DEVICES=1 npm run dev",
+      "console": "integratedTerminal",
+      "serverReadyAction": {
+        "pattern": "started server on .+, url: (https?://.+)",
+        "uriFormat": "%s",
+        "action": "debugWithChrome"
+      }
+    }
+  ]
+}
```

Comment:
We should make this repo a [multi-root workspace.](https://code.visualstudio.com/docs/editor/multi-root-workspaces) That way each folder can have its own settings but they can also share global options.

Reply:
@mdemoret-nv I added a global workspace, and local settings as needed per folder, let me know if it looks good